### PR TITLE
updated navbar

### PR DESCRIPTION
**Pull request by samgivian**
I updated the navbar to become responsive

### CODE REVIEW

File: src/components/NavBar/NavBar.js [hunk 1]
```diff
@@ -7,10 +7,13 @@ import DropDownButton from '../DropDownButton/DropDownButton';
 import DropDownMenu from '../DropDownMenu/DropDownMenu';
 import { useNavigate } from 'react-router-dom';
 
+
```
Comment (reginawang3495) [inline src/components/NavBar/NavBar.js:10]:
```suggestion
```

File: src/components/NavBar/NavBar.js [hunk 2]
```diff
@@ -7,10 +7,13 @@ import DropDownButton from '../DropDownButton/DropDownButton';
...
 const NavBar = () => {
     const [isMenuExpanded, setMenuExpanded] = useState(false);
     const navigate = useNavigate();
 
+
+
```
Comment (reginawang3495) [inline src/components/NavBar/NavBar.js:16]:
```suggestion
```

File: src/components/NavBar/NavBar.js [hunk 3]
```diff
@@ -21,17 +24,26 @@ const NavBar = () => {
...
                         text="Resources"
                         isExpanded={isMenuExpanded}
                         onClick={() => setMenuExpanded(prev => !prev)}
                     />
                 </div>
+                
+                <button className={isMenuExpanded ? 'active' : ''} id="hamburger" type="button" onClick={() => setMenuExpanded(prev => !prev)} aria-label="navigation menu" aria-expanded={isMenuExpanded} tabIndex="0">
+                    <span className= "bar" id="bar-one"></span>
+                    <span className= "bar" id="bar-two"></span>
+                    <span className="bar" id="bar-three"></span>
```
Comment (reginawang3495) [inline src/components/NavBar/NavBar.js:39]:
```suggestion
                    <span className="bar" id="bar-one"></span>
                    <span className="bar" id="bar-two"></span>
                    <span className="bar" id="bar-three"></span>
```

File: src/components/NavBar/NavBar.js [hunk 4]
```diff
@@ -21,17 +24,26 @@ const NavBar = () => {
...
+                </button>
             </div>
-            {isMenuExpanded ? <DropDownMenu resources={resources}/> : null}
+            {isMenuExpanded ? <DropDownMenu resources={resources} /> : null}
+
+
```
Comment (reginawang3495) [inline src/components/NavBar/NavBar.js:44]: you can do this!
```suggestion
            {isMenuExpanded && <DropDownMenu resources={resources} />}
```

File: src/components/DropDownMenu/DropDownMenu.css [hunk 5]
```diff
@@ -2,14 +2,15 @@
     background-color: #ffffff;
     box-shadow: 0px 2px #0000001b;
     width: 100%;
-    padding: 60px;
+    padding: 40px;
+ /*   padding-left: 40px;*/
```
Comment (reginawang3495) [inline src/components/DropDownMenu/DropDownMenu.css:6]:
```suggestion
```

**Review by reginawang3495 — COMMENTED** [review]
looks good to me ! just easy code cleanup + `&&` thoughts :)

I'd add in "related to #39" or "closes #39" in the PR summary so that Github connected your issue to your PR properly!
@doubleiis02 probably has more thoughts also :))

**Review by samgivian — COMMENTED** [review]
This shows 3 bars instead of the resource button on the navbar when the website is opened on phone.

File: src/components/NavBar/NavBar.js (hunk 4 above)
Comment (samgivian) [reply src/components/NavBar/NavBar.js]: Interesting. I didn't about short circuit evaluations.

**Review by reginawang3495 — DISMISSED** [review]
lgtm! waiting on @doubleiis02 review :)

File: src/components/NavBar/NavBar.js [hunk 6]
```diff
@@ -21,17 +21,24 @@ const NavBar = () => {
                         alt="acm logo"
                         onClick={() => navigate("/")} />
                 </div>
-                <div className="element-container">
+                
+                <div className="element-container resouce-btn">
```
Comment (doubleiis02) [inline src/components/NavBar/NavBar.js:25]: really minor but i think there's a typo for the class name lol should be `resource-btn` right? i see that the same typo is in the `NavBar.css` file

File: src/components/NavBar/NavBar.css [hunk 7]
```diff
@@ -1,18 +1,67 @@
...
+  transform: rotate(-45deg) translate(5px, -5px);
+}
+#hamburger:hover .bar{
+  background-color:#1E6CFF;
+}
+@media (max-width: 720px) {
+  #hamburger {
+    display: flex;
+  }
+  .resouce-btn{
```
Comment (doubleiis02) [inline src/components/NavBar/NavBar.css:64]: same as the comment below

**Review by doubleiis02 — CHANGES_REQUESTED** [review]
looks great!! very impressed with the hamburger animation :) i noticed one tiny typo but other than that you should be good to go! good job remembering the aria-label and aria-expanded

File: src/components/NavBar/NavBar.css (hunk 7 above)
Comment (samgivian) [reply src/components/NavBar/NavBar.css]: good point!

**Comment by reginawang3495** [comment]
Btw this isn’t compiling @samgivian ;0

**Review by doubleiis02 — APPROVED** [review]
LGTM! you can squash & merge now :)

**Pull request merged**